open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -217,6 +218,1600 @@ private ClassificationDef addTemplateClassification()
+        this.archiveBuilder.addTypeDefPatch(updateGraphStoreEntity());
+        this.archiveBuilder.addTypeDefPatch(updateLogFileEntity());
+        this.archiveBuilder.addTypeDefPatch(updateDatabaseEntity());
+        this.archiveBuilder.addTypeDefPatch(updateDatabaseServerClassification());
+        this.archiveBuilder.addTypeDefPatch(updateMetadataRepositoryEntity());
+        this.archiveBuilder.addTypeDefPatch(updateMetadataServerClassification());
+        this.archiveBuilder.addTypeDefPatch(updateRepositoryProxyClassification());
+        this.archiveBuilder.addTypeDefPatch(updateBusinessCapabilityEntity());
+        this.archiveBuilder.addTypeDefPatch(updateDataStoreEntity());
+        this.archiveBuilder.addTypeDefPatch(updateDataSourcePhysicalStatusAnnotationEntity());
+        this.archiveBuilder.addTypeDefPatch(updateEnterpriseAccessLayerEntity());
+        this.archiveBuilder.addTypeDefPatch(updateMetadataCollectionEntity());
+        this.archiveBuilder.addTypeDefPatch(updateExternalReferenceEntity());
+        this.archiveBuilder.addTypeDefPatch(updatePropertyFacetEntity());
+        this.archiveBuilder.addTypeDefPatch(updateCohortMemberEntity());
+        this.archiveBuilder.addTypeDefPatch(updateImplementationSnippetEntity());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyAdministrationPointClassification());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyDecisionPointClassification());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyEnforcementPointClassification());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyInformationPointClassification());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyRetrievalPointClassification());
+    }
+
+
+    /**
+     * Deprecate clashing properties and add new ones to replace them.
+     * @return the type def patch
+     * @see #updateSoftwareServerPlatformEntity()
+     */
+    private TypeDefPatch updateSoftwareServerPlatformEntity()
+    {
+        /*
+         * Create the Patch
+         */
+        final String typeName = "SoftwareServerPlatform";
+
+        TypeDefPatch  typeDefPatch = archiveBuilder.getPatchForType(typeName);
+
+        typeDefPatch.setUpdatedBy(originatorName);
+        typeDefPatch.setUpdateTime(creationDate);
+
+        /*
+         * Build the attributes
+         */
+        List<TypeDefAttribute> properties = new ArrayList<>();
+        TypeDefAttribute       property;
+
+        final String attribute1Name            = "type";
+        final String attribute1Description     = "Deprecated attribute. Use the platformType attribute to describe the type of software server platform.";
+        final String attribute1DescriptionGUID = null;
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute1Name,
+                attribute1Description,
+                attribute1DescriptionGUID);
+        property.setAttributeStatus(TypeDefAttributeStatus.DEPRECATED_ATTRIBUTE);
```

Comment:
Add "replacedBy" to guide reader to the new attribute.

Reply:
Done -- didn't see that on the example I'd copied from.

Reply:
it maybe that the one you copied did not have a replacement?